FAILURE: Build failed with an exception.

We noticed that the 25.2.2 build tools have been removed:

https://jcenter.bintray.com/com/android/tools/common/

Error below:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':@ptomasroos/react-native-idfa'.
> Could not resolve all files for configuration ':@ptomasroos/react-native-idfa:classpath'.
   > Could not find com.android.tools:common:25.2.2.
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/common/25.2.2/common-25.2.2.pom
         https://jcenter.bintray.com/com/android/tools/common/25.2.2/common-25.2.2.jar
     Required by:
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools.build:manifest-merger:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools.ddms:ddmlib:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools.analytics-library:shared:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools.analytics-library:tracker:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools:sdklib:25.2.2 > com.android.tools.layoutlib:layoutlib-api:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools:sdklib:25.2.2 > com.android.tools:dvlib:25.2.2
         project :@ptomasroos/react-native-idfa > com.android.tools.build:gradle:2.2.2 > com.android.tools.build:gradle-core:2.2.2 > com.android.tools.build:builder:2.2.2 > com.android.tools:sdklib:25.2.2 > com.android.tools:repository:25.2.2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 14s
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -31,8 +31,8 @@ allprojects {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 25)
-    buildToolsVersion safeExtGet('buildToolsVersion', '25.0.0')
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
@@ -43,7 +43,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:25.0.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'com.google.android.gms:play-services-ads:+'
     implementation 'com.facebook.react:react-native:+'
 }

